user-journey-tests/page-objects/checkAnswersPage.js

```diff
@@ -0,0 +1,39 @@
+import { Page } from './page.js'
+
+class CheckAnswersPage extends Page {
+  get checkAnswersTitle() {
+    return 'Check your answers before you continue your application'
+  }
+
+  get checkAnswersUrlPath() {
+    return 'origin/summary'
+  }
+
+  // Change links
+  get changeOnOrOffLink() {
+    return $('[data-testid="on-off-change-link"]')
```

Comment:
Are these globals?  I thought they needed to be imported!

Reply:
Nope I added them in myself